vpat 26: quickformat more detailed instructions

abaevbog opened the pull request:
I think this makes it a bit clearer as to what exactly is going on in the QF and what options are available.

Fixes: #4002

adomasven commented on chrome/locale/en-US/zotero/zotero.ftl:
```diff
@@ -468,9 +468,15 @@ toggle-preview =
+} Press { return-or-enter } to save edits to this citation. Press Escape to discard the changes and close the dialog.
+
+quickformat-aria-bubble = Item included in this citation. Press Down Arrow to open citation properties. { quickformat-general-instructions }
+quickformat-aria-input = Citation item search input. Type to search for an item to include in this citation. Then Press Tab to focus and navigate the list of matching items to add to this citation. { quickformat-general-instructions }
```
Maybe "Then Press Tab to focus and navigate the list of search results to add to this citation."?

abaevbog replied:
Sure, that sounds nice

adomasven commented:
> The citation selection process should provide clear, step-by-step instructions guiding users on how to search for, select, modify, and confirm citations, as well as how to add multiple citations if desired.

I didn't test this with a screen reader, but is "open citation properties" clear enough regarding "instructions on how to [...] modify [...] citations"? I guess that popover just has fields that are pretty self-explanatory?

abaevbog commented:
> I didn't test this with a screen reader, but is "open citation properties" clear enough regarding "instructions on how to [...] modify [...] citations"? I guess that popover just has fields that are pretty self-explanatory?

I changed it for "Item included in this citation. Press Down Arrow to open a dialog with citation properties to add details. " - that should indicate as to "why" you would want to open the popup. Also, I added the aria properties to make it clear that the properties popover is a dialog.

And then, yeah, as soon as it is opened, the "Page" (or whatever locator is selected) input is focused, and it gets announced as such, so it should be fairly clear (or at least clearer) what can be done.
